services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/S3TransferManager.java

```diff
@@ -503,33 +506,13 @@ static S3TransferManager.Builder builder() {
     interface Builder {
 
         /**
-         * Configuration values for the low level S3 client. The {@link S3TransferManager} already provides sensible
-         * defaults. All values are optional.
-         *
-         * @param configuration the configuration to use
+         * Low level S3 client that implements {@link S3AsyncClient}.The {@link S3TransferManager} already provides sensible
+         * default client. As of now only {@link S3CrtAsyncClient} supports concurrent execution of Transfer manager operations.
+         * @param s3AsyncClient Implementation of {@link S3AsyncClient}
```

Comment:
Can we clarify that the provided S3AsyncClient will not be closed when the transfer manager is closed?